src/main.py

```diff
@@ -401,6 +403,10 @@ def run_inner(options: Options, heartbeat: Callable[[], None]) -> List[int]:
         # Connect to network and create client threads and queues.
         net_conns: "List[Tuple[threading.Thread, Queue[Task]]]" = []
         if options.use_network:
+
+            from .net.client import start_client
+            from .net.core import ServerError, connect, enable_debug_mode
```

Comment:
These imports being here needs a comment, and the README needs to be updated to mention that the `pynacl` dependency is optional and doesn't work on Windows. And I do think we should have a try-except here with a nice error message, otherwise this optional support looks like a bug.

Reply:
on second thought I guess it's fine to skip the try-except, given instructions in the readme